### ISSUE TITLE
NDS v4.0.1 not available in OpenWrt

### ISSUE THREAD

**Issue by bawantha** [nodogsplash/nodogsplash#410]
Hi Tried to install NDS on OpenWrt 
```
opkg install nodogsplash
Unknown package 'nodogsplash'.
Collected errors:
 * opkg_install_cmd: Cannot install package nodogsplash.

```

**Comment by bluewavenet** [comment]
@bawantha 
```
opkg update
opkg install nodogsplash
```

**Comment by bawantha** [comment]
@bluewavenet  Yeah I typed `opkg update ` at first but it didn't work package is not availble

**Comment by bluewavenet** [comment]
It is available for OpenWrt 18.06, 19.07 and Snapshot, for all architectures.
There must be something wrong with your online connection or opkg setup.
Assuming you are still running on Rpi, you can download directly from:
http://downloads.openwrt.org/snapshots/packages/aarch64_cortex-a53/routing/nodogsplash_4.0.1-1_aarch64_cortex-a53.ipk or browse to the required architecture if you have other hardware.
